rpcs3/Emu/RSX/VK/vkutils/swapchain.hpp

```diff
@@ -586,8 +586,49 @@ namespace vk
 			VkSwapchainKHR old_swapchain = m_vk_swapchain;
 			vk::physical_device& gpu = const_cast<vk::physical_device&>(dev.gpu());
 
-			VkSurfaceCapabilitiesKHR surface_descriptors = {};
-			CHECK_RESULT(vkGetPhysicalDeviceSurfaceCapabilitiesKHR(gpu, m_surface, &surface_descriptors));
+			VkSurfaceCapabilities2KHR pSurfaceCapabilities = {};

```

Comment:
This method is uncomfortably long now, I'll have to refactor it later. Seems fine otherwise.